pkg/rest/writer.go

```diff
@@ -0,0 +1,87 @@
+package rest
+
+import (
+	"encoding/json"
+	"net/http"
+
+	osb "github.com/pmorie/go-open-service-broker-client/v2"
+)
+
+// writeResponse will serialize 'object' to the HTTP ResponseWriter
+// using the 'code' as the HTTP status code
+func (s *APISurface) writeResponse(w http.ResponseWriter, code int, object interface{}) {
+	data, err := json.Marshal(object)
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+
+	if s.EnableCORS {
+		//Allow CORS here By * or specific origin
+		w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
I'd prefer to see this value be configurable by the broker, instead of blanket allowing any origin.

Reply:
Are you okay with that in a follow-up?